packages/forms/src/Testing/TestsForms.php

```diff
@@ -401,4 +402,111 @@ public function assertFormFieldIsVisible(): Closure
             return $this;
         };
     }
+
+    public function assertWizardStepExists(): Closure
+    {
+        return function (int $step, string $formName = 'form'): static {
+            /** @phpstan-ignore-next-line  */
+            $this->assertFormExists($formName);
+
+            /** @var ComponentContainer $form */
+            $form = $this->instance()->{$formName};
+
+            /**
+             * @var Wizard $wizard
+             */
+            $wizard = $form->getComponent(fn ($component) => $component instanceof Wizard);
+            Assert::assertTrue(count($wizard->getChildComponents()) >= $step, "Wizard does not have a step $step");
+
+            return $this;
+        };
+    }
+
+    public function assertWizardCurrentStepIs(): Closure
+    {
+        return function (int $step, string $formName = 'form'): static {
+            /** @phpstan-ignore-next-line  */
+            $this->assertFormExists($formName);
+
+            /** @var ComponentContainer $form */
+            $form = $this->instance()->{$formName};
+
+            /**
+             * @var Wizard $wizard
+             */
```

Comment:
Please condense this down to one line, like the other `@var` comments for consistency